src/setup/deployment/packaging/test/zip_test.go

```diff
@@ -0,0 +1,39 @@
+package packaging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"os"
+	"stellar/setup/building"
+	"stellar/setup/deployment/packaging"
+	"testing"
+)
+
+type ZipTestSuite struct {
+	suite.Suite
+}
+
+func (s *ZipTestSuite) SetupSuite() {
+	if err := os.Chdir("../../../.."); err != nil {
+		log.Fatal("Failed to change to /src directory ")
+	}
+}
+
+func (s *ZipTestSuite) TestGenerateServerlessZipArtifactsPython() {
```

Comment:
Could you add an additional check to see if the final zip size is 50MB?

Reply:
I modified the test to include a assertion that the ZIP is within ~0.1MB of 50MB